static_src/stores/login_store.js

```diff
@@ -7,25 +7,44 @@ import BaseStore from './base_store.js';
 import { loginActionTypes } from '../constants.js';
 
 
-class LoginStore extends BaseStore {
+export class LoginStore extends BaseStore {
   constructor() {
     super();
     this.subscribe(() => this._registerToActions.bind(this));
+    // TODO this should probably be false, but we need to account for the
```

Comment:
Yeah, that sounds like a good idea. I'm fine with using `null` and just being explicit in the getters (`null ===true`, `null==false` both fail) but using enums would make this much more explicit.